### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [RealFredChan/colmar#4]
https://github.com/RealFredChan/colmar/blob/b0b24a8746fc9ae2f01a2aa1746238a49894c6a4/index.html#L71

Alt text should be used for all images.  This helps the visually impaired through the screen readers, plus your site will not pass validation without it.
